packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CaptureOutput.swift

```diff
@@ -72,8 +71,8 @@ extension AVCapturePhotoOutput: CapturePhotoOutput {
     return self
   }
 
-  func connection(with mediaType: AVMediaType) -> FLTCaptureConnection? {
-    guard let connection: AVCaptureConnection = connection(with: mediaType) else { return nil }
-    return FLTDefaultCaptureConnection(connection: connection)
+  func connection(with mediaType: AVMediaType) -> CaptureConnection? {
+    let connection: AVCaptureConnection? = connection(with: mediaType)
+    return connection
```

Comment:
nit: can be 1 line

Reply:
if you need infer the type, you can do `return AVCaptureConnection.connection(with: mediaType)`